README.md

```diff
@@ -101,13 +101,13 @@ Firstly, if I violate the rights of major music publishers, the takedown request
 
 Secondly, musescore.com is not a simple music sharing website. The authors of sheet music must transcript and rearrange the original songs to sheets, not just copying files from somewhere else to musescore.com. As a result, the licensing should focus on the rights of transcription/rearranging to the sheet music authors, instead of the rights of sharing the music on some websites.
 
-Thirdly, the copyright ownership of contents on musescore.com is not clear. Not every non-public-domain songs on musescore.com is owned by major music publishers. There are many small music publishers and independent songwriters; Songs might be licensed under free licenses like Creative Commons. Also, there are many authors who created their own songs and posted the sheet music on musescore.com. Does musescore.com pay to those authors?
+Thirdly, the copyright ownership of contents on musescore.com is not clear. Not every non-public-domain song on musescore.com is owned by major music publishers. There are many small music publishers and independent songwriters; Songs might be licensed under free licenses like Creative Commons. Also, there are many authors who created their own songs and posted the sheet music on musescore.com. Does musescore.com pay to those authors?
 
-If we can't see proof that musescore.com really pays licensing fee to the copyright owners, we may think it is just an excuse to get profit from stealing.
+If we can't see proof that musescore.com really pays license fees to the copyright owners, we may think it is just an excuse to get profit from stealing.
 
-> you illegaly use our private API with licensed music content.
+> you illegally use our private API with licensed music content.
```

Comment:
```suggestion
> you illegaly [sic] use our private API with licensed music content.
```

This line was from [Chistyakov's email](https://github.com/Xmader/musescore-downloader/issues/5#issue-562094186)